src/devices/Nmea0183/tests/Ais/AidToNavigationReportMessageTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Iot.Device.Nmea0183.Ais;
+using Iot.Device.Nmea0183.AisSentences;
+using Shouldly;
+using Xunit;
+
+namespace Iot.Device.Nmea0183.Tests.Ais
+{
+    public class AidToNavigationReportMessageTests : MessageTestBase
+    {
+        [Fact]
+        public void Should_parse_message()
+        {
+            const string sentence = "!AIVDM,1,1,,B,ENk`sR9`92ah97PR9h0W1T@1@@@=MTpS<7GFP00003vP000,2*4B";
```

Comment:
If tests are copied from anywhere please makes sure to add a comment on top

Reply:
Good catch, forgot to add the comment to the tests. Done now.